lib/services.template.ejs

```diff
@@ -267,8 +267,10 @@ module
       props.forEach(function(name) {
         self[name] = load(name);
       });
-      this.rememberMe = undefined;
-      this.currentUserData = null;
+      if ( !load('rememberMe') ){
+        this.rememberMe = undefined;
+        this.currentUserData = null;
+      }
```

Comment:
Since `rememberMe` is already in `props`, the check can be written as `if (!this.rememberMe)`, unless I am missing something?

Also I don't understand why `currentUserData` is cleared conditionally - IMO it is unrelated to `rememberMe`.

When put together, I think you only need to remove `this.rememberMe = undefined` from the original code.

```diff
  props.forEach(function(name) {
    self[name] = load(name);
  });
- this.rememberMe = undefined; 
  this.currentUserData = null;
```